### ISSUE TITLE
[version_0.5]Home/ Office address can not remove 

### ISSUE THREAD

**Issue by Joeyd321** [Telenav-Search/search-android-demo-app#11]
Home Office address only can add and edit, can not delete .

**Comment by arambaghdasintellias** [comment]
That flow doesn't exist in UI design.

**Comment by Joeyd321** [comment]
> That flow doesn't exist in UI design.

Yes, in UX design , it doesn't exist , but for an user , delete function is necessary . 
You can confirm with product manager, if she/he agree that do not need delete function, you can close it with by design .

**Comment by Toothpickliu** [comment]
> > That flow doesn't exist in UI design.
> 
> Yes, in UX design , it doesn't exist , but for an user , delete function is necessary . You can confirm with product manager, if she/he agree that do not need delete function, you can close it with by design .

Agree. will add this delete function in UX flow.

**Comment by Toothpickliu** [comment]
> > > That flow doesn't exist in UI design.
> > 
> > 
> > Yes, in UX design , it doesn't exist , but for an user , delete function is necessary . You can confirm with product manager, if she/he agree that do not need delete function, you can close it with by design .
> 
> Agree. will add this delete function in UX flow.

Here is the figma link about the updated UX flow, please check it https://www.figma.com/file/khwN0kYvlm0y5i408dCXfy/Search-SDK?node-id=219%3A11452
